mm2src/mm2_main/src/lp_ordermatch.rs

```diff
@@ -1401,6 +1414,17 @@ impl<'a> TakerOrderBuilder<'a> {
         self
     }
 
+    /// When a new [TakerOrderBuilder::new] is created, it sets [SWAP_VERSION] by default.
+    /// However, if user has not specified in the config to use TPU V2,
+    /// the TakerOrderBuilder's swap_version is changed to legacy.
+    /// In the future alls users will be using TPU V2 by default without "use_trading_proto_v2" configuration.
+    pub fn use_trading_proto_v2(mut self, use_trading_proto_v2: bool) -> Self {
+        if !use_trading_proto_v2 {
+            self.swap_version = 1;
```

Comment:
we should use a const like LEGACY_SWAP_VERSION (indicating current version this code supports)

Reply:
Current version this code/repo supports is `const SWAP_VERSION`
`use_trading_proto_v2:false` means user didnt enable TPU

reused const fn https://github.com/KomodoPlatform/komodo-defi-framework/pull/2324/commits/53d4b403121ae1614b18c9390ba32b935a05fe35